feature: copy selection with line numbers

roman-szul-red opened the pull request:
feature: copy selection with line numbers

variar committed:
ci: add oracle 7

variar committed:
chore: add better stats for release downloads

roman-szul-red committed:
feature: copy selection with line numbers

tomasz-blaszczak-red commented:
@roman-szul-red, fine in overall, but keeping in mind that our changes are going to be published in origin repo I am raising priority on keeping code convention, mostly I mean to add a space after opening bracket and before closing one, e.g.:
```
(const AbstractLogData *logData)
  vs
( const AbstractLogData *logData )
```

jima80525 committed:
Update DOCUMENTATION.md (#503)

Just adding a note about the .conf file extension for exported files.

roman-szul-red committed:
formatting corrected. line number before selection

roman-szul-red commented:
> @roman-szul-red, fine in overall, but keeping in mind that our changes are going to be published in origin repo I am raising priority on keeping code convention, mostly I mean to add a space after opening bracket and before closing one, e.g.:
> 
> ```
> (const AbstractLogData *logData)
>   vs
> ( const AbstractLogData *logData )
> ```

A quality fist punch. Yep, clang-format plugin was disabled. Code reformatted.

roman-szul-red committed:
fix: common format single selection, partial and multi line selection

roman-szul-red committed:
Merge branch 'variar:master' into clipboard_copy_with_line_numbers

roman-szul-red committed:
Export filters fix file extension (#531)

* fix: ensure exported filters have conf extension

* reformat the code

roman-szul-red committed:
fix: removed code duplication

Commit:
feat: Add option to save only selected lines to file (#236)

variar committed:
build: fix uchardet repo url

variar committed:
build: update CPM

variar committed:
ci: disable codesigning on Win, to much hassle to support

variar committed:
ci: update runner versions and openssl

variar committed:
ci: fix openssl url again

variar committed:
build: try fix uchardet url again

variar committed:
Revert "build: try fix uchardet url again"

This reverts commit b0a8b068bcd5331c9785cf2766fc1ab998ddc775.

variar committed:
ci: disable centos7 build

variar committed:
ci: fix tbb copying to release on win

variar committed:
build: add compatibility with qt 6.4

variar committed:
ci: copy redist from new place

variar committed:
ci: don't override memory allocator on Windows in CI builds

Abay11 committed:
fix: append extension to export file if not present (#536)

* fix: append extension to export file if not present

* style: replace tabs with spaces

---------

Co-authored-by: Alim Abaev <alim.abaev@axxonsoft.dev>

variar committed:
ci: bump version

nathaniel-brough committed:
Update simdutf version (#552)

The latest simdutf includes various security and performance improvements.

Abay11 committed:
feat: Add a button allowing to quickly clean up the Filter textbox (#544) (#556)

Co-authored-by: Alim Abaev <alim.abaev@axxonsoft.dev>

Abay11 committed:
feat: Multiple line selection via shortcuts Shift+Up, Shift+Down (#542)

Co-authored-by: Alim Abaev <alim.abaev@axxonsoft.dev>

variar committed:
feat: change cancel search icon back to monochrome (#554)

tomasz-blaszczak-red committed:
feat: Reflect search field on check-boxes in Predefined Filters (#527)

User input from search line field is reflected in Predefined Filters
menu, i. e. any predefined filter is marked as checked if it is a part
of user input from the search line field.

variar committed:
fix: update predefined filter selection also on startup (#527)

franciszek-koltuniuk-red committed:
feat: Move recent files to a separate submenu (#528)

* Move recent files to a separate submenu

Changes:
 - move the list of recent files to a dedicated submenu
 - added an option to configure the number of recent files to list
 - added option to clean up the recent files list

variar committed:
fix: make recent files PR compatible with qt6 and move settings to File tab (#528)

variar committed:
docs: update dev stats links [skip ci]

Abay11 committed:
fix: return deleted shortcut ReplaceSearch (#557)

Co-authored-by: Alim Abaev <alim.abaev@axxonsoft.dev>

nowhszh committed:
fix: select lines up/down shortcut no action name (#559)

nowhszh committed:
Add i18n for Klogg (#558)

* feat: Add internationalization support

* feat: Add Chinese tranlate

* feat: Use tr() on the strings to be translated and translate them

* chore: Add qt5 linguist for dockfile

* feat: update src/app/i18n/zh-CN.ts

---------

Authored-by: marvin <huaishuizhenhuai@foxmail>

variar committed:
tr: use capital L for language group

variar committed:
ci: update github actions

GermanAizek committed:
refactor: Added const ref params, remove std::move() with const ref, replace push_back on emplace_back (#554)

variar committed:
Merge branch 'master' into clipboard_copy_with_line_numbers
